src/components/Button/BaseButton.tsx

```diff
@@ -197,10 +199,14 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
     const getButtonLoader = (): JSX.Element =>
       loading && <Loader classNames={styles.loader} size={getLoaderSize()} />;
 
-    const getButtonIcon = (): JSX.Element => (
+    const getButtonIcon = (position = 'left'): JSX.Element => (
```

Comment:
can we create a different method for getting the second icon, will be cleaner

Reply:
Will do